src/darjeeling/transformation/base.py

```diff
@@ -20,12 +20,6 @@ def to_replacement(self) -> Replacement:
         """Converts a transformation into a source code replacement."""
         ...
 
-    @property
-    @abc.abstractmethod
-    def line(self) -> FileLine:
```

Comment:
Ah. The `line` method is part of the `Transformation` interface, which we implement an an abstract base class in Python. We should keep the original code here.

Reply:
I'm unsure how to fix the problem:
```TypeError: Can't instantiate abstract class ReplaceLine with abstract methods line``` then.